apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -550,6 +550,7 @@ export const database: NavMenuConstant = {
       name: 'Postgres Guides',
       url: undefined,
       items: [
+        { name: 'Inspecting your Postgres database', url: '/guides/database/inspect' },
```

Comment:
perhaps there is a clearer title for this one -  he goal in the title is to give the reader a "hook", so that they know why they would read it. 

Perhaps something like "Monitoring performance, indexes, and Postgres internals"? (or something similar)